app/models/draftMovements/GetDraftMovementsSearchOptions.scala

```diff
@@ -74,7 +81,7 @@ object GetDraftMovementsSearchOptions extends Logging {
           index <- intBinder.bind("index", params).getOrElse(Right(DEFAULT_INDEX))
           searchTerm <- stringBinder.bind("searchTerm", params).map(_.map(Some(_))).getOrElse(Right(None))
           draftHasErrors <- booleanBinder.bind("draftHasErrors", params).map(_.map(Some(_))).getOrElse(Right(None))
-          destinationTypes <- destinationTypeBinder.bind("destinationType", params).map(_.map(Some(_))).getOrElse(Right(None))
+          destinationTypes <- destinationTypeBinder.bind("destinationTypes", params).map(_.map(Some(_))).getOrElse(Right(None))
```

Comment:
I think this probably would make sense to stay as singular of `destinationType` because it will be repeated in the URL multiple times for each one. So each would be singular.